### DISCUSSION TITLE
Default module for users

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#15106] [How to] [answered]
Cannot find setting to allow each user having different "Default" module. Is it exists?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://gitdeveloper.yetiforce.com/index.php?module=Users&parent=Settings&view=Detail&record=6

![image](https://user-images.githubusercontent.com/10198654/130597099-fa5fb418-7ae0-472f-8550-9034301dd647.png)

**Comment by s2b-git** [comment]
this is related to default search module. There is no column in users table related to default module.
This setting changing entry of default_search_module column :-)
Was looking for it too

**Reply by mariuszkrzaczkowski** [reply]
you're right, I mistook the mechanisms, it can only be in the file
https://doc.yetiforce.com/code/classes/Config-Main.html#property_default_module

**Reply by s2b-git** [reply]
may I suggest a wish list element? 
"Add Default Landing Module per user" 
 :-)
